### ISSUE TITLE
"Edit Favorite Stops": Pagination should be centered

### ISSUE THREAD

**Issue by janpio** [umts/pvta-multiplatform#419]
... instead of `padding-left`ed. Tappable area should probably also be a bit bigger.

**Comment by mboneil10** [comment]
Is there an ionic/mobile app style guide you're referring to, @janpio?

**Comment by janpio** [comment]
No, this just looks odd non-centered. iPhone 5S looks quite different than a iPhone 6 Plus for example.

**Comment by mboneil10** [comment]
Ah, okay. Thank you. I agree that the `padding` looks a bit odd and could be centered instead. As for the tappable area, it seems to be the same size as the stop checkboxes.

**Comment by akaplo** [comment]
@mboneil10 if you need help with this lmk. FYI we use a [library for pagination](https://github.com/michaelbromley/ngx-pagination).  They have an option for us to override their styling entirely but keep their functionality (it works pretty well) - perhaps it's something we should consider.

**Comment by mboneil10** [comment]
Okay, I can do that.

**Comment by mboneil10** [comment]
I'm hesitant on overriding styling. But since it's only a library I don't feel that bad about it.

**Comment by akaplo** [comment]
The library is built under the assumption that users will be overriding.

**Comment by mboneil10** [comment]
Seems like we can't easily override the styling on this library. Issue is on standby.

**Comment by mboneil10** [comment]
Maybe #457 will fix this.
